modules/social_features/social_album/src/Plugin/Field/FieldWidget/SocialAlbumOptionsSelectWidget.php

```diff
@@ -24,6 +24,22 @@
  */
 class SocialAlbumOptionsSelectWidget extends OptionsSelectWidget {
 
+  /**
+   * The visibility options mapping.
+   *
+   * The associative array where keys are node options and values are the
+   * corresponding post options.
+   *
+   * @see field.storage.node.field_content_visibility.yml
+   * @see field.storage.post.field_visibility.yml
+   */
+  const VISIBILITY_MAPPING = [
+    '0' => 'community',
```

Comment:
@OH17L could you explain why do we have both 0 and 2 for the community here?

Reply:
Because these values were gotten from allowed values [here](https://github.com/goalgorilla/open_social/blob/10.0.x/modules/social_features/social_post/config/install/field.storage.post.field_visibility.yml#L14).

Reply:
Ok, then it's clear.